modules/i18n/src/main/java/com/devonfw/module/i18n/locale/api/I18n.java

```diff
@@ -0,0 +1,20 @@
+package com.devonfw.module.i18n.locale.api;
+
+/**
+ * Interface for getting JSON using locale and filter.
+ *
+ */
+
+public interface I18n {
+
+  /**
+   * Gets the JSON string for specified locale and filter
+   *
+   * @param locale locale
+   * @param filter
+   * @return Json String
+   * @throws Exception
+   */
+  String getResourceObject(String locale, String filter) throws Exception;
```

Comment:
Anti-Pattern: Do not declare API that throws Exception. We also have the best-practice in devon4j *not* to use checked exceptions:
https://github.com/devonfw/devon4j/blob/develop/documentation/guide-exceptions.asciidoc#exception-principles